doc_source/related/dev.md

```diff
@@ -0,0 +1,409 @@
+Libuavcan Library Developer Guide {#LibDevGuide}
+====================================================
+
+> This is a guide for developers of libuavcan itself. If you are a user of libuavcan you don't need to read this, but hey; Thanks for using our software.
+
+-----------------------------------------------------
+
+> And now...on to the sausage making!
+
+## Build System
+
+We require [cmake 3.6](https://cmake.org/cmake/help/v3.6/) or greater because that's what Ubuntu 18.04 ships with and Ubuntu 18.04 is the reference operating system we as our generic build and test environment. At the time of this writing we have only tested the GNU Makefile generator but there's no reason why ninja shouldn't work either.
```

Comment:
```suggestion
We require [cmake 3.6](https://cmake.org/cmake/help/v3.6/) or greater because that's what Ubuntu 18.04 ships with and Ubuntu 18.04 is the reference operating system we use as our generic build and test environment. At the time of this writing we have only tested the GNU Makefile generator but there's no reason why ninja shouldn't work either.
```